service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ReviewCMOControllerAboutToSubmitTest.java

```diff
@@ -99,12 +104,62 @@ void shouldSealPDFAndAddToSealedCMOsListWhenJudgeApprovesOrder() throws Exceptio
             .status(APPROVED)
             .build();
 
+        assertThat(State.ISSUE_RESOLUTION).isNotEqualTo(responseData.getState());
         assertThat(responseData.getDraftUploadedCMOs()).isEmpty();
         assertThat(responseData.getSealedCMOs())
             .extracting(Element::getValue)
             .containsExactly(expectedSealedCmo);
     }
 
+    @Test
+    void shouldUpdateStateToIssueResolutionWhenNextHearingIssueResolutionAndCmoDecisionIsSendToAllParties()
+        throws Exception {
+        DocumentReference convertedDocument = testDocumentReference();
+        DocumentReference sealedDocument = testDocumentReference();
+
```

Comment:
these could be added to the top of the class like private CaseManagementOrder cmo = buildCMO(); and then the two    

given(documentConversionService.convertToPdf(cmo.getOrder())).willReturn(convertedDocument);
given(documentSealingService.sealDocument(convertedDocument)).willReturn(sealedDocument);

could be removed from the below two tests and added in a @BeforeEach? 

Happy if you want to leave. :-)